docs/quickstart.rst

```diff
@@ -1,8 +1,132 @@
 .. include:: global.rst.inc
-.. highlight:: bash
 .. _quickstart:
 
 Quick Start
 ===========
 
 This chapter will get you started with Borg-Import.
+
+Prerequisites
+-------------
+
+Before using borg-import, you need:
+
+1. **BorgBackup installed**: borg-import requires the ``borg`` command to be available in your PATH.
+    Install it following the `BorgBackup installation guide <https://borgbackup.readthedocs.io/en/stable/installation.html>`_.
```

Comment:
Strictly taken, the user has to use the version selector there and select precisely the version they are using.